WooCommerce.xcworkspace/contents.xcworkspacedata

```diff
@@ -22,6 +22,9 @@
    <FileRef
       location = "group:Fakes/Fakes.xcodeproj">
    </FileRef>
+   <FileRef
+      location = "group:Observables">
+   </FileRef>
```

Comment:
nit: we probably don't want to commit these diffs since the Observables framework was removed earlier?